docs/src/content/pages/reader-api.mdoc

```diff
@@ -1,35 +1,62 @@
 ---
 title: Reader API
 summary: >-
-  The Reader API is a Node.js API that lets you read Keystatic content from your
-  file system.
+  The Reader API is a Node.js API that lets you read Keystatic content from a storage of your own choice.
 ---
-The Reader API is a Node.js API that lets you *read* Keystatic content from your file system.
+The Reader API is a Node.js API that lets you *read* Keystatic content from a storage of your own choice.
+The storage can be any local directory / GitHub repository, and does not need to be the same as the one defined in the Keystatic config.
 
 {% aside icon="⚠️" %}
 The code can only run on the server, and not in the browser.
 {% /aside %}
 
 ## Usage
 
-First, import the `createReader` function, as well as your Keystatic config file:
+{% aside icon="⚠️" %}
+There is currently no support for creating readers for `cloud` storage.
+{% /aside %}
+
```

Comment:
```suggestion
```

Reply:
I don't mind leaving this out but I feel people who use `cloud` will ask questions either way.

Reply:
Creating a reader with a cloud config works totally fine?

Reply:
I don't see a `createCloudReader` that's why I thought it doesn't work. Not that I've looked into how `cloud` works though.